src/components/game/playZone/enterWord/soloEnterWord.tsx

```
@@ -0,0 +1,92 @@
+import useSoloGame from '@/hooks/useSoloGame';
+import { useCountActions } from '@/providers/storeProvider/countStoreProvider';
+import { useGameActions, useGameState } from '@/providers/storeProvider/gameStoreProvider';
+import { useWordActions } from '@/providers/storeProvider/wordStoreProvider';
+import { enterWordSchema } from '@/schema/enterWordSchema';
+import { checkWordIsValid } from '@/utils/word/checkWordValid';
+
+import type { z } from 'zod';
+
+type EnterWordInput = z.infer<typeof enterWordSchema>;
+
+interface Props {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)